src/unix/udp.c

```diff
@@ -660,6 +754,34 @@ int uv_udp_set_membership(uv_udp_t* handle,
   }
 }
 
+
+int uv_udp_set_source_membership(uv_udp_t* handle,
+                                 const char* multicast_addr,
+                                 const char* interface_addr,
+                                 const char* source_addr,
+                                 uv_membership membership) {
+  struct sockaddr_in mcast_addr4;
+  struct sockaddr_in src_addr4;
+  struct sockaddr_in6 mcast_addr6;
+  struct sockaddr_in6 src_addr6;
+
+  int err = uv_ip4_addr(multicast_addr, 0, &mcast_addr4);
```

Comment:
style: we separate variable declaration from assignment.

Reply:
fixed